imap/cyr_ls.c

```diff
@@ -182,7 +182,7 @@ static void long_list(struct stat *statp)
 
     strftime(datestr, 13, datefmt, localtime(&(statp->st_ctime)));
 
-    printf("%c%c%c%c%c%c%c%c%c%c %lu %-8s %-8s % 10ld %s ",
+    printf("%c%c%c%c%c%c%c%c%c%c %ju %-8s %-8s % 10ld %s ",
```

Comment:
`j` isn't portable here.  Please use the `PRIuMAX` macro (may need to include `<inttypes.h>` if it's not already included)

Something like this (untested):

```
    printf("%c%c%c%c%c%c%c%c%c%c %" PRIuMAX "u %-8s %-8s % 10ld %s ",
```

Reply:
Thanks! Did not know about `PRIuMAX`

Reply:
There's one of these `PRI[iu][size]` portable printf width macros for each of the `[u]int[size]_t` types, and a set of corresponding `SCN...` ones for use with scanf (which are usually the same as the printf one, but can theoretically be different).  I think they were introduced in C99?